### DISCUSSION TITLE
Table color/background - 5.2 -> 5.3

### DISCUSSION THREAD

**Discussion by chrissearle** [twbs/bootstrap#38727] [Q&A]
Struggling with a change from 5.2.3 to 5.3.0 where it comes to table background color and text color.

Trying to find out the _simplest_ way to get the same output with 5.3.0 as I had in 5.2.3 (am no CSS-person - so it may well be me doing something stupid)

Using 5.2.3 I can set the background for an entire table with a bg-* class and the text color within the table using text-* on the table tag.  This meant that I could get a table to merge within a surrounding container in terms of color.

For example - let's use success as background and white as text - just to have something that shows the issue:

In 5.2.3 I could state `<table class="table text-white bg-success">` 

[5.2.3 example](https://htmlpreview.github.io/?https://gist.githubusercontent.com/chrissearle/680a3f39ff9e4995ba35beea594d2590/raw/2dac7b86f826df484c9c3ceeb76ae9dd4c1bc190/bs52.html)

Updating to 5.3.0 with the new color stuff - those are ignored.

I've tried setting the class to the new table-* - that does take effect - but gives me a transparent over white color and the text color is still ignored.

5.3.0 example has two tables - first with no changes - second with `<table class="table table-success text-white">`

[5.3.0 example](https://htmlpreview.github.io/?https://gist.githubusercontent.com/chrissearle/680a3f39ff9e4995ba35beea594d2590/raw/2dac7b86f826df484c9c3ceeb76ae9dd4c1bc190/bs53.html)

Code for both visible on:

https://gist.github.com/chrissearle/680a3f39ff9e4995ba35beea594d2590

**Comment by MullenStudio** [comment]
I just updated one of my projects yesterday and have exactly same issue. There are much more color options from bg-* than table-* and bg-* could cover the child table which is useful when I don't want the table to be stand out of the background. I would like to know if this change is by design or a bug.
